scripts/src/java/org/oppia/android/scripts/lint/BUILD.bazel

```diff
@@ -11,9 +11,12 @@ kt_jvm_library(
         "AndroidLintRunner.kt",
     ],
     visibility = ["//scripts:oppia_script_binary_visibility"],
+    # Added to to push the JRE guava version up the dependency chain
```

Comment:
Nit: Can be expanded to explain more. Something like "# Export guava-jre to resolve version conflicts in the dependency chain. This ensures downstream targets use the JRE version instead of Android version"